kernel/core/strings/stringfunctions.hpp

```diff
@@ -29,6 +29,8 @@
 // define the "End of transmission block" as string separator
 #define NEWSTRING (char)23
 
+extern Language _guilang;

```

Comment:
```suggestion
extern Language _lang;

```
Works with both, but I'd like to have both instances separated.